src/packages/cli/src/init/template.ts

```diff
@@ -74,8 +74,19 @@ export const makeDatabase = (projectName: string, backends: Backend[]) => {
 	},
 };`;
 
+	const liteDriverImport = `import { SqliteDriver } from '@mikro-orm/sqlite';`;
+	const liteConnection = `export const liteConnection = {
+	connectionManagerId: 'sqlite',
+	mikroOrmConfig: {
+		driver: SqliteDriver,
```

Comment:
Is this also the first time we're supporting SQLLite?

Reply:
yeah!